### PR TITLE
Manage yum::versionlock with concat

### PR DESCRIPTION
**Pull request by jpoittevin**
To avoid multiple lines when updating a version

### CODE REVIEW

File: manifests/versionlock.pp [hunk 1]
```diff
@@ -18,37 +17,32 @@
...
+  }
 
-      if ($ensure == exclude) or ($ensure == absent) {
-        file_line { "versionlock.list-!${name}":
-          ensure => $ensure,
-          line   => "!${_name}",
-          path   => $path,
-        }
+  case $ensure {
+    present,exclude: {
```
Comment (bastelfreak) [inline manifests/versionlock.pp]: we need singleqoutes here

File: manifests/versionlock.pp [hunk 2]
```diff
@@ -18,37 +17,32 @@
...
+      concat::fragment { "yum-versionlock-${name}":
+        content => "${line_prefix}${line}\n",
+        target  => $yum::plugin::versionlock::path,
       }
     }
-
+    absent:{
```
Comment (bastelfreak) [inline manifests/versionlock.pp]: we need singlequotes here

File: metadata.json [hunk 3]
```diff
@@ -11,6 +11,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.2.0 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 1.1.1 < 2.0.0"
```
Comment (bastelfreak) [inline metadata.json]: why lower than 2.0.0?

**Comment by bastelfreak** [comment]
Hi, thanks for the PR. Can you please take a look at the comments I made and/or and the failed travis run?

File: metadata.json (hunk 3 above)
Comment (jpoittevin) [inline metadata.json]: Good question... it was just a copy/paste from another plugin. I will remove this.

**Pull request merged**